crates/modelardb_test/Cargo.toml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 The ModelarDB Contributors
+# Copyright 2025 The ModelarDB Contributors
```

Comment:
Are the dates in the PR purposely updated? I do not recall us doing this in the past @CGodiksen and @chrthomsen.

Reply:
Fixed